packages/react/src/components/Authenticator/withAuthenticator.tsx

```diff
@@ -2,13 +2,13 @@ import { Authenticator, AuthenticatorProps } from './Authenticator';
 
 export type WithAuthenticatorOptions = Omit<AuthenticatorProps, 'children'>;
 
-export function withAuthenticator(
-  Component,
+export function withAuthenticator<T>(
```

Comment:
Can we use something other than `T` here? I'm not a fan of single letter generic type params. Maybe something like:

```suggestion
export function withAuthenticator<ComponentProps>(
```